helm/grid-intensity-exporter/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: "v1"
+description: "A Prometheus exporter for understanding the carbon intensity of compute."
+home: "https://github.com/thegreenwebfoundation/grid-intensity-exporter"
+name: "grid-intensity-exporter"
+version: "0.1.0"
```

Comment:
The rest of this is the Helm chart itself. I've tried to keep it simple but as usual there is a lot of YAML 😅